src/test/java/step1/StringTest.java

```diff
@@ -0,0 +1,60 @@
+package step1;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+
+	@Test
+	public void test1(){
+		String string = "1,2";
+		String[] strings = string.split(",");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?